.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Scala Steward: Reformat with scalafmt 3.5.8
+400c1be6df0495e80143357a08a6e9605553f489
```

Comment:
This is super duper awesome 😎

Reply:
Oh nice! I didn't even know this was a thing!